text/0000-template-changes.md

```diff
@@ -0,0 +1,61 @@
+# Meta
+[meta]: #meta
+- Name: RFC template changes
+- Start Date: 2020-02-03
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+As a response to the process of approving [RFC 0013](0013-app-layer-metadata-source.md) which ended up requiring spec changes, this RFC proposes amending the template to include an optional (non-binding) section to document spec changes that fall out of an RFC.
+
+# Motivation
+[motivation]: #motivation
+
+Improve the process of moving RFCs into spec and implementation.
+
+# What it is
+[what-it-is]: #what-it-is
+
+[0000-template.md](../0000-template.md) gets a new section:
+
+```diff
+
+# Prior Art
+[prior-art]: #prior-art
+
+Discuss prior art, both the good and bad.
+
++
++ # Spec. Changes (OPTIONAL)
++ [spec-changes]: #spec-changes
++
++ Does this RFC entail any proposed changes to the specifications (either buildpack or platform) or extensions?
```

Comment:
```suggestion
+ Does this RFC entail any proposed changes to the specifications or extensions?
```

Distribution too. Might as not provide a comprehensive list for the sake of future proofing

Reply:
Agree being more general here would be good to future-proof.

Reply:
What about something like a checklist? This would help people easily think through all the potential candidates. Some changes apply to multiple specs.

```
If this RFC proposes changes to a specification, what specification?

- [ ] platform
- [ ] buildpack
- [ ] distribution
- [ ] extension: ___________
```